### ISSUE TITLE
Problem for display subgroups

### ISSUE THREAD

**Issue by petitbx** [rpi-virtuell/rw-bp-group-hierarchy#1]
Hello, 

I have a problem with the plugin to display subgroups on my wordpress.
I use the plugin with Symfony2 and the Bundle EkinoWordpressBundle. 
All the mapping is good but I have a problem with the hierarchy.js and I don't know where.
If my principal group have a subgroup, he have the [+] and when I click on it, it just only refresh the page.

I you have some ideas of my problem.

Thanks

 petitbx

**Comment by staude** [comment]
See you errors in the browser console?

**Comment by petitbx** [comment]
No, I just have a problem with the hierarchy.js, don't find the function bp_init_objects but when I copy/paste it on the File that's doesn't work.
